core-rust/state-manager/src/metrics.rs

```diff
@@ -226,3 +237,59 @@ impl<T: MetricLabel> MetricLabel for &T {
         T::prometheus_label_name(self)
     }
 }
+
+// Concrete types used for metric labels:
+
+impl MetricLabel for ComponentAddress {
+    type StringReturnType = String;
+
+    fn prometheus_label_name(&self) -> Self::StringReturnType {
+        self.to_hex()
+    }
+}
+
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum ConsensusRoundResolution {
+    Successful,
+    Fallback,
+    Gap,
```

Comment:
_(note from reviewer)_

I am open to any renames of the label names and values in this PR.

Reply:
We generally call them `missed proposals` so maybe:
- Successful
- MissedAndFallback
- MissedAndTimeout
?